hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCatalog.java

```diff
@@ -247,17 +249,28 @@ public void testReplaceTxnBuilder() throws Exception {
 
   @Test
   public void testCreateTableWithOwner() throws Exception {
+    createTableAndVerifyOwner(
```

Comment:
Why are these mixed into a single test method rather than two test methods? I think this makes it harder to follow